auth-api/src/auth_api/services/invitation.py

```diff
@@ -46,11 +52,11 @@ def as_dict(self):
         return obj
 
     @staticmethod
-    def create_invitation(invitation_info: dict, user_id):
+    def create_invitation(invitation_info: dict, user_id, user):
```

Comment:
If we're passing in `user` now, can we just get `user.id` from that instead of passing in `user_id` separately?